src/manage_nsfs/manage_nsfs_help_utils.js

```diff
@@ -180,6 +180,7 @@ bucket list [flags]
 Flags:
 --wide                                                (optional)        Print the additional details for each bucket
 --name <string>                                       (optional)        Filter the list based on the provided bucket name
+--force                                               (optional)        Forcefully delete bucket if the bucket is not empty
```

Comment:
Please move this to `BUCKET_FLAGS_DELETE` (currently it is in `BUCKET_FLAGS_LIST`).